ui/app/utils/constants/capabilities.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import apiPath from 'vault/utils/api-path';
+
```

Comment:
Can you add a comment here to clarify what sudo paths indicate? Does it have anything to do with Sudo permissions?

Reply:
I don't know exactly what they are for I just moved them from the service to this constants file since it seemed like a better place for them. I would assume they are paths that require sudo permission maybe but would rather not comment without knowing for sure.

Reply:
I looks like those were originally added to the capabilities model 7 years ago.